CONTRIBUTING.md

```diff
@@ -41,28 +41,12 @@ A typical contributor workflow looks like this:
       `yarn lint` to be 100% safe.
     - TypeScript lint errors can often be automatically fixed by TSLint. Run lint fixes with `yarn lint-fix`.
 1. Submit a Pull Request on GitHub and fill out the template.
+    - :warning: __DO NOT enable CircleCI for your fork of Blueprint.__ Our build
+      will run on your fork when you open a PR. You can run relevant parts of
+      the build locally by inspecting the config or
```

Comment:
"by inspecting the config or"... or what?

Reply:
heh thanks